### PR TITLE
Fixed console output

### PR DESCRIPTION
**Pull request by mkoshurnikov**
- Fixed console output in UpdateProductSample.
- Added using directive.

### CODE REVIEW

File: retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@
 // Update product from a catalog using Retail API
 
 using Google.Cloud.Retail.V2;
+using Google.Protobuf.WellKnownTypes;
```
Comment (amanda-tarafa) [inline retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs:19]: I don't think you need this? What am I missing?

File: retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2021 Google Inc. All Rights Reserved.
+﻿// Copyright 2022 Google Inc. All Rights Reserved.
```
Comment (amanda-tarafa) [inline retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs:1]: Please revert, we don't update the copyright year when updating existing samples.

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]

File: retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs (hunk 1 above)
Comment (mkoshurnikov) [reply retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs:19]: we paste code during tutorial step, UpdateMask field in updateProductRequest variable

File: retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs (hunk 2 above)
Comment (mkoshurnikov) [reply retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs]: done

**Review by amanda-tarafa — APPROVED** [review]
LGTM, thanks.

**Comment by amanda-tarafa** [comment]
I'll wait till #1830 is fixed before running the tests for this one and merging. Thanks.

**Pull request merged**